client/services/FeedbackGeneratorService.js

```diff
@@ -208,11 +207,42 @@ tie.factory('FeedbackGeneratorService', [
           }
         }
       );
-      feedback.appendCodeParagraph(fixedErrorString);
+
+      // TODO: Will need to adjust depending on the language
+      var feedbackString = _getHumanReadableRuntimeFeedback(
+          fixedErrorString, 'python');
+      if (feedbackString === null) {
+        feedback.appendTextParagraph(
+            "Looks like your code had a runtime error" + inputClause +
+            ". Here's the trace:");
+        feedback.appendCodeParagraph(fixedErrorString);
+      } else {
+        feedback.appendTextParagraph(feedbackString);
+      }
       return feedback;
     };
 
     /**
+     * Based on passed in error string, will generate the appropriate,
+     * informative feedback to be appended to the overall submission feedback.
+     *
+     * @param {string} errorString
+     * @param {string} language
+     * @returns {string | null} Text to be appended to feedback.
+     */
+    var _getHumanReadableRuntimeFeedback = function(errorString, language) {
+      var result = null;
+      if (language === 'python') {
+        RUNTIME_ERROR_FEEDBACK_MESSAGES.python.forEach(function(check) {
```

Comment:
Why not just RUNTIME_ERROR_FEEDBACK_MESSAGES[language].forEach(function(check) {...